sonar-plugin-api/src/main/java/org/sonar/api/server/ws/WebService.java

```diff
@@ -636,6 +636,7 @@ public String toString() {
     private boolean required = false;
     private boolean internal = false;
     private Set<String> possibleValues = null;
+    private Integer maximumValues = null;
```

Comment:
name is confusing. We don't know if it's the max value allowed or the maximum number of value allowed. 
Suggestions: `maximumNumberOfValues`, `maximumValueCount`, `valueCountLimit`, `maxNbValues`, `maxNumberOfValues`.
It's in the API so it's better to get it right